include/zephyr/net/lwm2m.h

```diff
@@ -559,6 +560,20 @@ int lwm2m_swmgmt_install_completed(uint16_t obj_inst_id, int error_code);
 
 #endif
 
+#if defined(CONFIG_LWM2M_EVENT_LOG_OBJ_SUPPORT)
+
+/**
+ * @brief Set callback to read log data
+ *
+ * The callback will be executed when the LWM2M read operation gets called
+ * on the corresponding object.
+ *
+ * @param[in] cb A callback function for handling the read event.
+ */
+void lwm2m_event_log_set_read_log_data_cb(lwm2m_engine_get_data_cb_t cb);
```

Comment:
Do we really need a custom function for this, couldn't the application simply use `lwm2m_engine_register_read_callback()` to register a read callback?